R/pkg/R/utils.R

```diff
@@ -440,19 +446,23 @@ processClosure <- function(node, oldEnv, defVars, checkedFuncs, newEnv) {
         # Namespaces other than "SparkR" will not be searched.
         if (!isNamespace(func.env) ||
               (getNamespaceName(func.env) == "SparkR" &&
-              !(nodeChar %in% getNamespaceExports("SparkR")))) {  # Only include SparkR internals.
+              !(nodeChar %in% getNamespaceExports("SparkR")))) {
```

Comment:
Hmm yes - the `(` in `getNameSpace` could be aligned with the previous line I guess. However when we have nested conditions I actually think its more readable to align the inner condition like a new statement (with 2 spaces) as it is done here.